packages/iTwinUI-react/src/core/Tile/Tile.tsx

```diff
@@ -150,9 +192,12 @@ export const Tile = (props: TileProps) => {
           'iui-new': isNew,
           'iui-selected': isSelected,
           'iui-actionable': isActionable,
+          [`iui-${status}`]: !!status,
+          'iui-loading': isLoading,
         },
         className,
       )}
+      aria-disabled={isDisabled}
       tabIndex={isActionable ? 0 : undefined}
```

Comment:
Probably should prevent focusing the tile when it's disabled. It's not a good user experience, but it will be consistent with our components.
```suggestion
tabIndex={isActionable && !isDisabled ? 0 : undefined}
```